src/main/java/com/example/petbuddybackend/entity/care/CareStatusesHistory.java

```diff
@@ -0,0 +1,26 @@
+package com.example.petbuddybackend.entity.care;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.ZonedDateTime;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity
+@Getter
+@Setter
+public class CareStatusesHistory {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private ZonedDateTime createdAt;
```

Comment:
Dodałbym `@PreUpdate` ustawiający to na `ZonedDateTime.now()`

Reply:
w sumie to nwm czy nie dac tylko prepersista bez PreUpdate, bo tak naprawde zadnego z pol i tak nie mozna edytowac

Reply:
A racja, mi w sumie też o PrePersist chodziło, pomyliłem sie. Skoro już przy okazji fragment kodu zpushowałem i też się ze mną zgadzasz że to do wywalenia to już to zrobiłem